README.md

```diff
@@ -1,33 +1,38 @@
 # API Example
 
-Example apps using Climate's API.
+Example app using Climate's [FieldView API](https://dev.fieldview.com).
 
 ## Setup
 
 1. Install python 3.6+.
 
-    ```bash
-    # if you use Mac OS X and brew, this can be done with:
-    brew install python3
-    ```
+```bash
+# if you use Mac OS X and brew, this can be done with:
+brew install python3
+```
 
 2. Make a virtual environment, activate it, and install dependencies.
 
-    ```bash
-    python3 -m venv api-example
-    source api-example/bin/activate
-    pip install -r requirements.txt
-    ```
+```bash
+python3 -m venv api-example
+source api-example/bin/activate
+pip install -r requirements.txt
+```
 
-3. Set the following environment variables to the values provided by Climate:
+3. Set the following environment variables (or hardcode them in `main.py`)
+to the values provided to you by Climate:
 
-    ```bash
-    export CLIMATE_API_ID="partner-fooco"
-    export CLIMATE_API_SECRET="azbq56fpadhnt8oukoeani2a4w"
-    export CLIMATE_API_KEY="partner-fooco-216b9875-0158-4142-1ab2-7c3bdbd6a2157"
-    ```
+```bash
+export CLIMATE_API_ID="partner-fooco"
```

Comment:
do we need a check here to see if these env vars exist and only set when necessary

Reply:
at the moment, this isn't a script that gets run. rather its something the user would enter manually